src/App.jsx

```
@@ -1,204 +1,204 @@
-        </div>
-
+    <main className="min-h-screen bg-zinc-950 text-white font-sans flex flex-col items-center justify-center p-4 relative antialiased overflow-hidden">
+      <div className="absolute top-0 right-0 p-4 z-20">
         <button
           onClick={() => setIsSettingsOpen(true)}
-          className="ml-auto relative z-10 text-[#808080] hover:text-white/80 px-3 py-1.5 border-2 border-dashed border-[#808080]/30 rounded-[10px] hover:border-white/40 opacity-70 hover:opacity-90 transition-all duration-300 text-sm font-[Arial,sans-serif] focus:outline-none cursor-pointer"
+          className="p-3 rounded-full text-zinc-400 hover:text-white hover:bg-zinc-800/50 transition-all duration-200"
+          aria-label="Ouvrir les paramètres"
         >
-          Paramètres
+          <Settings className="w-6 h-6" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)